src/frontend-samples/component-gallery/CommonComponentTools/ComponentContainer.tsx

```diff
@@ -20,12 +23,14 @@ export class ComponentContainer extends React.Component<{ data: ComponentExample
   }
 }
 
+// These are the possible attributes of each of the components being displayed
 export interface ComponentExampleProps {
   title: string;
   description?: string;
   content: React.ReactNode;
 }
 
+// This stores and formats a single component, along with the corresponding title and description
```

Comment:
Is "stores" the right word?  Maybe "formats a single component, along with its title and description, and adds them to the DOM".  Or maybe "formats the UI elements for single component along with its title and description"

Reply:
I was trying to convey that it was a container for the component and it's corresponding information. I think mentioning adding them to the DOM clarifies that, so I'm gonna go with that